src/app/templates/balances-changes-view/layout.tsx

```diff
@@ -42,7 +42,7 @@ const BalancesChangesViewRow = memo<BalancesChangesViewRowProps>(({ chain, symbo
         !allCollectibles && (
           <span className="text-font-num-12 text-grey-1 ml-3 whitespace-nowrap">
             <InFiat
-              volume={volume}
+              volume={volume.absoluteValue()}
```

Comment:
Are you sure these changes should be applied for the previews of dApps transactions too? I'm not